.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--
+# Simple Summary
+This policy allows to write out rewards to complete required tasks. Completed tasks are payed by the Escrow council to the claiming member.
+
+# How to create a new bounty?
+
+1. Create an issue with bounty description and bounty and draft tags in an appropriate repository.
+   [Bounty template] [Example]
+   If the bounty spans across multiple repositories, consider splitting it in a smaller per-repo bounties if possible.
+   The bounty definition should be SMART (Specific, Measurable, Achievable, Relevant, Time bound)
+2. Submit proposal via the bounty form: http://bounty.leapdao.org/viewform
+3. Add the bounty to the bounties board: https://github.com/orgs/leapdao/projects/6
+4. If no objections for 2 days, the bounty is considered approved
+
+# Bounty sizes
+XS 200
+S  350
+M  550
+L  900
+XL 1400
+
+# Pair programming
+If 2 people claim the bounty together, the payout increases by 1.5x.
+
+# Bounty Challenge
+If bounty has no progress for 4 days, then anyone can challenge the bounty. Once the bounty is challenged, the worker has 3 more days to deliver some progress on the bounty (gardener can make exception and extend period to 5 days 1x, unless gardener is also working). If worker fails to make progress within period, the bounty is up for grabs by anyone.
+
+# Payout
+Only bounties from the current sprint(if any) are payed out. Unless you are newbie (never earned a bounty before), then you get 1 exception.
+-->
+# Bounty
+
+## Scope
+-
+-
+-
+
+## Deliverables
+
+## Gain for the project
+
+### Publicly visible delivery
```

Comment:
```suggestion
### Publicly visible delivery
<!-- ideally, each bounty should has at least a part of delivery which is publicly visible. This could be a blog post, tweet, documentation or website update etc. LeapDAO Slack is not considered public. -->

```